### PR TITLE
chore: rework qb expressions to css modules

### PR DESCRIPTION
**Pull request by uladzimirdev**
[epic](https://github.com/metabase/metabase/issues/50983)

### Description

migrate emotion to css modules in `query_builder/components/expressions`
2/?

### How to verify

Expression Editor is the same as in stats

### CODE REVIEW

File: frontend/src/metabase/css/core/colors.module.css [hunk 1]
```diff
@@ -100,6 +100,7 @@
     var(--mb-color-tooltip-background) 50%,
     #000
   );
+  --mb-color-syntax-parameters: var(--mb-base-color-mango-30);
```
Comment (uladzimirdev) [inline frontend/src/metabase/css/core/colors.module.css:103]: https://metaboat.slack.com/archives/C0645JP1W81/p1733315783925849?thread_ts=1733146244.214589&cid=C0645JP1W81

**Review by ranquild — APPROVED** [review]

**Comment by uladzimirdev** [comment]
UI is broken, I'll fix it

File: frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
+.ArgumentsGrid {
```
Comment (kamilmielnik) [inline frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css:1]: :x: "< Custom Expression" should be gray

image

File: frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  margin: 1rem 0;
```
Comment (kamilmielnik) [inline frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css:2]: :x: Arguments are now `rgb(227, 89, 94)` but used to be `rgb(239, 140, 140)`

image

File: frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  display: grid;
```
Comment (kamilmielnik) [inline frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css:3]: :x: Color of numbers has changed
image

**Review by kamilmielnik — CHANGES_REQUESTED** [review]

File: frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css (hunk 3 above)
Comment (uladzimirdev) [reply frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css:2]: it's fine, we agreed to use this color https://metaboat.slack.com/archives/C0645JP1W81/p1733315783925849?thread_ts=1733146244.214589&cid=C0645JP1W81

File: frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css (hunk 4 above)
Comment (uladzimirdev) [reply frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css:3]: image

6eb9d18

File: frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css (hunk 2 above)
Comment (uladzimirdev) [reply frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText/ExpressionEditorHelpText.module.css:1]: fixed  6eb9d18

image

File: frontend/src/metabase/query_builder/components/expressions/ExpressionWidgetHeader.module.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
+.Header {
+  color: var(--mb-color-text-medium);
+  padding: 1rem 1.5rem;
+  border-bottom: 1px solid var(--mb-color-border);
+  display: flex;
+  align-items: center;
+
+  .HeaderButton {
+    color: var(--mb-color-text-light);
+  }
```
Comment (kamilmielnik) [inline frontend/src/metabase/query_builder/components/expressions/ExpressionWidgetHeader.module.css:10]: Text in `.HeaderButton` should be blue on hover.

Comment (uladzimirdev) [reply frontend/src/metabase/query_builder/components/expressions/ExpressionWidgetHeader.module.css]: should be fixed

https://github.com/user-attachments/assets/1ef787d7-6d43-4eca-988a-42a6e778d10b

**Review by kamilmielnik — APPROVED** [review]

**Pull request merged**